jenkins/scripts/perf/submit.py

```
@@ -513,13 +535,14 @@ def main():
         srun_args_lines.append("--container-env=pytestCommand")
     else:
         # Disaggregated (e2e or gen_only).
-        base_worker_env_vars = (
+        base_prefix = (
             f"FLASHINFER_JIT_DIR=/tmp/flashinfer_jit_cache_\\${{SLURM_LOCALID}} "
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)